specify/specify7
# Implement delete all notifications API

Fixes #4494 



### Checklist

      and self-explanatory (or properly documented)

### Testing instructions

1. Get notifications to pop up (ex: merge agents)
2. Click "Clear All" and verify that all notifications are cleared
